Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift

```diff
@@ -14,11 +14,15 @@ struct BasicAppSettingsView: View {
     private let legal = App.configuration.legal
     private let app = App.configuration.app
 
+    @State var addOwnerIsActive = false
```

Comment:
We usually move @State on separated line

Reply:
changed everywhere